From a8bc9c413482ada01bfa513797ac79c2ec8e3524 Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Wed, 30 Jul 2025 16:30:26 +0200
Subject: [PATCH 74/94] Revert "sysctl: Removed unused variable"

ia64 still uses this, so we keep that.

Otherwise we get:
```
kernel/sysctl.c:1504:36: error: 'unaligned_dump_stack' undeclared here (not in a function); did you mean 'trace_dump_stack'?
 1504 |                 .data           = &unaligned_dump_stack,
      |                                    ^~~~~~~~~~~~~~~~~~~~
      |                                    trace_dump_stack
```
...during build.

This reverts commit 39dac316f09ae5a0930878d2cae8aea113648b5a.
---
 include/linux/sysctl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/sysctl.h b/include/linux/sysctl.h
index 92e9146b1104..30bcbc59d12d 100644
--- a/include/linux/sysctl.h
+++ b/include/linux/sysctl.h
@@ -243,6 +243,7 @@ int do_proc_douintvec(const struct ctl_table *table, int write,
 		      void *data);
 
 extern int unaligned_enabled;
+extern int unaligned_dump_stack;
 extern int no_unaligned_warning;
 
 #else /* CONFIG_SYSCTL */
-- 
2.25.1

